### ISSUE TITLE
Annotation submission review page GO and ECO terms should link out to AmiGO browser

### ISSUE THREAD

**Issue by lreiser** [tair/toastdos-back#336]
What:
On the submission review page the expected behavior is for the ECO and GO terms to be hyperlinks to the corresponding entry in the Evidence Ontology or Gene Ontology/AmiGO browser.

Why:
This linking useful for both the submitter and the curator roles as a way to easily view the definition and relationships of terms to ensure that the correct/best choice of term has made.

How:
The base URL for linking out to ECO is
http://www.evidenceontology.org/browse/#
e.g 
http://www.evidenceontology.org/browse/#ECO:0005801

the URL for linking out to AmiGO (for GO and PO terms)
http://amigo.geneontology.org/amigo/term/_**GO:0000257**_#display-lineage-tab

where the GO or PO term ID is the **_variable_**

Example:
Current version
![screenshot_1565](https://user-images.githubusercontent.com/14040923/53069219-02e4e100-3490-11e9-97bd-7cf5af721d37.png)
Old Version
![screenshot_1566](https://user-images.githubusercontent.com/14040923/53069224-0710fe80-3490-11e9-87d5-83c8829e8b4b.png)

**Comment by JKingSE** [comment]
For the random ECO and GO terms we tried, these links worked.

**Comment by lreiser** [comment]
This looks good.. only one slight issue
One thing I noticed is the protein-protein interaction annotations attempt to provide a link to IPI evidence code  in addition to the ECO term (see the IPI linked in Green)
![screenshot_1684](https://user-images.githubusercontent.com/14040923/56540398-78673f00-651d-11e9-8c41-defd01af1d1f.png)

If you look at the previous version there is only the link to the ECO term
![screenshot_1685](https://user-images.githubusercontent.com/14040923/56540429-8cab3c00-651d-11e9-8732-6879c31fa50f.png)

**Comment by tberardini** [comment]
@lreiser  What do you think about the link to GO?  Right now, it looks like the link is to the Inferred Tree View, e.g. http://amigo.geneontology.org/amigo/term/GO:0033436#display-lineage-tab.  Do you think we should link to the top of the page (with name and def) instead? http://amigo.geneontology.org/amigo/term/GO:0033436

**Comment by lreiser** [comment]
The behavior is consistent with the implementation from last year. 
@tberardini  do you want to change the logic?
I'd have to go back and think about why we opted for the expanded tree view (perhaps to encourage people to look for related terms) vs looking to the definition per se.

**Comment by tberardini** [comment]
Not necessarily.  It just happened that with the term I had selected, the term name was not visible in the view I could see in my screen so I thought that could be confusing to a first time user.  This does not happen in all cases, of course.

term =  AUG codon-amino acid adaptor activity

<img width="819" alt="image" src="https://user-images.githubusercontent.com/4998871/56600380-6634e180-65ad-11e9-90f8-519a16b6bb77.png">

Let's leave as is right now as one can of course scroll up and down.

**Comment by lreiser** [comment]
pls fix the issue where the 'IPI' evidence code is linking-otherwise looks good.

**Comment by qwertyflagstop** [comment]
Need to create a special parser for IPI links similar to ECO and GO, moving to to-do.
